packages/angular/projects/ui-angular/src/lib/components/authenticator/components/authenticator/authenticator.component.ts

```diff
@@ -126,7 +126,7 @@ export class AuthenticatorComponent
 
         this.hasInitialized = true;
       }
-    }).unsubscribe;
```

Comment:
`AuthenticatorService.subscribe` now always returns `unsubscribe` directly, so `Authenticator` no longer need to access `.unsubscribe` to get the `unsubscribe` function.